docs/index.rst

```diff
@@ -42,6 +42,7 @@ Getting Started
    :maxdepth: 1
 
    install
+   What's New in astropy 1.3? <http://docs.astropy.org/en/stable/whatsnew/1.3.html>
```

Comment:
no need to add a static url, it's enough to say ``whatsnew/1.3``, sphinx will figure out the rest

![screen shot 2017-06-22 at 17 57 12](https://user-images.githubusercontent.com/6788290/27446121-56c590d0-5774-11e7-8b14-c0d1473bbf1b.png)